api/v3/Payment.php

```diff
@@ -134,8 +134,8 @@ function civicrm_api3_payment_create(&$params) {
   }
   // Get contribution
   $contribution = civicrm_api3('Contribution', 'getsingle', array('id' => $params['contribution_id']));
-  if ($contribution['contribution_status'] != 'Partially paid') {
-    throw new API_Exception('Please select a contribution which has a partial payment');
+  if (!in_array($contribution['contribution_status'], array('Partially paid', 'Pending'))) {
```

Comment:
I don't know the answer to this but - is the contribution_status field ever returned translated - if so this would break. Ideally I guess we would have a test which changed the language to see if things still work.